### ISSUE TITLE
Hyphen in app ID results in bad apk package

### ISSUE THREAD

**Issue by kgolding** [fyne-io/fyne-cross#116]
Having a hyphen in the app-id result in apk package that will not install with the error "There was a problem while parsing the package" when trying to install on the hadset, and the following error when installing in dev mode using `adb install ...`

Failure [INSTALL_PARSE_FAILED_BAD_PACKAGE_NAME: Invalid manifest package: bad character '-']

### To Reproduce:
Steps to reproduce the behaviour:
1. In FyneApp.toml in the [Details] section set `ID` to a string including a hyphen `-`
2. Run `fyne-cross adroid`
3. Transfer APK file to device and click to install

### Device and debug info:

<summary>Device info</summary>

 - **OS:** Android
 - **Version:** 12
 - **Go version:** 1.1.8
 - **fyne-cross version:** 1.2.0
 - **Fyne version:** 2.1.4

**Comment by Jacalz** [comment]
Indeed. Android does not support `-` in appIDs and they are generally discouraged in AppStream metadata as well (see https://github.com/ximion/metainfocreator/issues/12) I wonder if this issue perhaps makes more sense in `fyne-io/fyne` than here in `fyne-cross`?

**Comment by andydotxyz** [comment]
Agreed, feel free to move as it is `fyne package` that does this work.
